Content.Shared/CCVar/CCVars.Game.cs

```diff
@@ -239,6 +239,94 @@ public static readonly CVarDef<bool>
+
+        /// <summary>
+        /// A contact email to be sent along with the request. Required by IPIntel
+        /// </summary>
+        public static readonly CVarDef<string> IPIntelEmail =
+            CVarDef.Create("game.ipintel_contact_email", string.Empty, CVar.SERVERONLY | CVar.CONFIDENTIAL);
+
+        /// <summary>
+        /// The URL to IPIntel to make requests to. If you pay for more queries this is what you want to change.
+        /// </summary>
+        public static readonly CVarDef<string> IPIntelBase =
+            CVarDef.Create("game.ipintel_baseurl", "https://check.getipintel.net", CVar.SERVERONLY);
+
+        /// <summary>
+        /// The flags to use in the request to IPIntel, please look here for more info. https://getipintel.net/free-proxy-vpn-tor-detection-api/#optional_settings
+        /// Note: Some flags may increase the chances of false positives and request time. The default should be fine for most servers.
+        /// </summary>
+        public static readonly CVarDef<string> IPIntelFlags =
+            CVarDef.Create("game.ipintel_flags", "b", CVar.SERVERONLY);
+
+        /// <summary>
+        /// Maximum amount of requests per Minute. For free you get 15.
+        /// </summary>
+        public static readonly CVarDef<int> IPIntelMaxMinute =
+            CVarDef.Create("game.ipintel_request_limit_minute", 15, CVar.SERVERONLY);
+
+        /// <summary>
+        /// Maximum amount of requests per Day. For free you get 500.
+        /// </summary>
+        public static readonly CVarDef<int> IPIntelMaxDay =
+            CVarDef.Create("game.ipintel_request_limit_daily", 500, CVar.SERVERONLY);
+
+        /// <summary>
+        /// How long to store results in the cache before they must be retrieved again in days.
+        /// </summary>
+        public static readonly CVarDef<int> IPIntelCacheLength =
+            CVarDef.Create("game.ipintel_cache_length", 7, CVar.SERVERONLY);
+
+        /// <summary>
+        /// Amount of playtime in minutes to be exempt from an IP check. 0 to search everyone. 5 hours by default.
+        /// <remarks>
+        /// Trust me you want one.
+        /// </remarks>>
+        /// </summary>
+        public static readonly CVarDef<int> IPIntelExemptPlaytime =
+            CVarDef.Create("game.ipintel_exempt_playtime", 300, CVar.SERVERONLY);
+
+        /// <summary>
+        /// Rating to reject at. Anything equal to or higher than this will reject the connection.
+        /// </summary>
+        public static readonly CVarDef<float> IPIntelBadRating =
+            CVarDef.Create("game.ipintel_bad_rating", 0.8f, CVar.SERVERONLY);
+
+        /// <summary>
+        /// Rating to send an admin warning over, but not reject the connection. Set to 0 to disable
+        /// </summary>
+        public static readonly CVarDef<float> IPIntelAlertAdminWarnRating =
+            CVarDef.Create("game.ipintel_alert_admin_warn_rating", 0f, CVar.SERVERONLY);
```

Comment:
0.8 seems to be too low. A maintainer of the project was concerned as they got 0.97.

Ipintel recommends automated action on 0.99. However, I will put up a warning rating as well by default.

I thought 0.8 would be fine since its what tg had (well at least had on their repo)